modules/sr/robot/output_frequency_limiter.py

```diff
@@ -15,6 +15,10 @@ def can_change(self) -> bool:
         now = self._webot.getTime()
         diff = now - self._last_change
 
+        # allow infinite changes within a timestep
```

Comment:
I think it would be useful to explain why this is ok. Assuming my understanding is correct, something like this might be more useful to future readers:
```suggestion
        # Changes within a timestep always all happen during the same
        # render cycle and therefore cannot contribute to additional strobing.
        # It's therefore safe to allow several changes within the same timestep.
```

Reply:
That is much better wording